### PR TITLE
Tableau Exchange branding

### PR DESCRIPTION
**Pull request by Craigfis**

### CODE REVIEW

File: docs/trex_publish.md [hunk 1]
```diff
@@ -62,12 +62,12 @@ In addition to the requirements for all extensions to ensure security and usabil
  * Declaring data access requirements
  * Providing a URL that customers can use to get support for your the extension
  
- Dashboard extensions that appear in the gallery must also:
+ Dashboard extensions that appear in the Exchange must also:
 
 * Follow the Design Guidelines for Dashboard Extensions (user interaction and style guidelines for user interface elements).
 * Ensure that the information in the extension manifest file (`.trex`) matches the content that you will publish in the Extension Gallery. For example, the `name`, `description` fields are used to populate the name and description fields in the Extension Gallery.  The icon you use in the manifest should also be the icon that is used in the gallery. You need to provide a 280x280 pixel `.png` version of the icon.
```
Comment (bcantoni) [inline docs/trex_publish.md:68]: Couple more instances of Extension Gallery here if you want to change them

**Comment by bcantoni** [comment]
Craig I notice the branch conflicts here - this branch might need rebasing again from main, and maybe that'll fix some of those typos like 'the the' which I think you fixed previously.

File: docs/trex_publish.md (hunk 1 above)
Comment (seanmakesgames) [reply docs/trex_publish.md]: done

**Comment by seanmakesgames** [comment]
@Craigfis what brian said. (John and I just found this PR)

Get the conflicts sorted and we'll merge it up!

**Comment by Craigfis** [comment]
If only I knew how to rebase my branch in this github web UI...

**Comment by d45** [comment]
@Craigfis  Hey, Craig. I can fix the branding from Gallery to Exchange. I need to apologize, as I had this on my plate to correct for the v1.8 release. It's probably a lot easier for me to get this change. It looks like you were trying to merge directly with the `main` branch. We like to make the fixes in the `development` branch first. If it's all right with you, I'll open a new PR to fix this in both branches.  @bcantoni @seanmakesgames

**Comment by bcantoni** [comment]
I'll close this one in favor of Dave's #435

**Pull request closed without merging**